classes/PolicyPositions.php

```diff
@@ -56,11 +56,12 @@ class PolicyPositions {
      * @param int      $limit    The number of policies to limit the list to.
      */
 
-    public function __construct(Policies $policies, Member $member, $limit = NULL)
+    public function __construct(Policies $policies, Member $member, $limit = NULL, $summaries = array())
```

Comment:
Rather than having to pass NULL in to all the ones without a limit, perhaps the constructor could be (Policies, Member, Options) with Options being a hash that can supply limit or summaries?